pkgs/servers/alps/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, buildGoModule, fetchFromSourcehut }:
+
+buildGoModule rec {
+  pname = "alps";
+  version = "d4c35f3c3157bece8e50fd95f2ee1081be30d7ae";
+
+  src = fetchFromSourcehut {
+    owner = "~migadu";
+    repo = "alps";
+    rev = "v${version}";
+    sha256 = "0000000000000000000000000000000000000000000000000000";
```

Comment:
You are missing a top-level entry and you can run nix-build locally to get this.

Reply:
I was just looking into docs to figure this out, thank you! I'll push the changes up after I test.

Reply:
Done, thanks!